doc/workshop/exercises/analysis/meshing/meshing.rst

```diff
@@ -32,7 +39,7 @@ CloudCompare.
 .. note::
 
     This example will be using the output of using :ref:`workshop-clipping`
-    exercise, specifically using ``stadium.las`` file.  Please produce that
+    exercise, specifically using ``stadium.copc.las`` file.  Please produce that
```

Comment:
copc needs to be a .laz, not .las file. Make sure that any exercise conversion that has changed to using COPC is making LAZ files.